Dockerfile

```diff
@@ -17,7 +17,16 @@ ENV FACTORIO_AUTOSAVE_INTERVAL=2 \
     VERSION=0.13.8 \
     FACTORIO_SHA1=fd835ec0987a525ddc92acb003d9c75745261bb2 \
     FACTORIO_WAITING=false \
-    FACTORIO_MODE=normal
+    FACTORIO_MODE=normal \
+    FACTORIO_SERVER_NAME="Factorio Server" \
+    FACTORIO_SERVER_DESCRIPTION="" \
+    FACTORIO_SERVER_MAX_PLAYERS="" \
+    FACTORIO_SERVER_VISIBILITY="hidden" \
+    FACTORIO_USER_USERNAME="" \
+    FACTORIO_USER_PASSWORD="" \
+    FACTORIO_USER_TOKEN="" \
+    FACTORIO_SERVER_GAME_PASSWORD="" \
+    FACTORIO_SERVER_VERIFY_IDENTITY="false"
 
```

Comment:
Instead of checking defaults and doing a check later on to determine if "not default" why not set empty here and set default later in the smart_launch.sh.